packages/publish-views/lib/ValidationError.js

```diff
@@ -0,0 +1,43 @@
+import { rdf, sh } from '@tpluscode/rdf-ns-builders'
+import $rdf from 'rdf-ext'
+import clownface from 'clownface'
+
+export class ValidationError extends Error {
```

Comment:
This is similar to the error class from barnard package, but I needed something different, which would combine multiple reports